.github/workflows/release.yml

```
@@ -2,202 +2,276 @@ name: Release
+              }
+            }
+
+      - name: Update release summary
+        run: |
+          echo "## Release Summary" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+          echo "Version: ${{ needs.release.outputs.new_release_version }}" >> $GITHUB_STEP_SUMMARY
+          echo "Release URL: ${{ steps.release_notes.outputs.release_url }}" >> $GITHUB_STEP_SUMMARY
+          echo "npm URL: https://www.npmjs.com/package/mcp-console-automation/v/${{ needs.release.outputs.new_release_version }}" >> $GITHUB_STEP_SUMMARY
+
+  publish-mcp-registry:
```

actionlint (1.7.8)

could not parse as YAML: did not find expected key

(syntax-check)